### PR TITLE
Available to use last tilt version

### PR DESCRIPTION
**Pull request by Fudoshiki**
sass-rails working nice with tilt 2.0.x versions

### CODE REVIEW

File: sass-rails.gemspec [hunk 1]
```diff
@@ -17,7 +17,7 @@ Gem::Specification.new do |s|
   s.add_dependency 'sass',            '~> 3.1'
   s.add_dependency 'sprockets-rails', '>= 2.0', '< 4.0'
   s.add_dependency 'sprockets',       '>= 2.8', '< 4.0'
-  s.add_dependency 'tilt',            '~> 1.1'
+  s.add_dependency 'tilt',            '>= 1.1'
```
Comment (rafaelfranca) [inline sass-rails.gemspec]: Please change to `'>= 1.1', `< 3``

**Comment by Fudoshiki** [comment]
@rafaelfranca, can You merge this?

**Comment by jaredbeck** [comment]
Is there a release of sass-rails with this change yet?  Thanks!

**Comment by rafaelfranca** [comment]
No.

**Pull request merged**